Firebase/Auth/Source/Public/FIRAuth.h

```diff
@@ -426,11 +427,13 @@ NS_SWIFT_NAME(Auth)
                             completion:(nullable FIRAuthDataResultCallback)completion;
 
 /** @fn signInWithCredential:completion:
-    @brief Convenience method for `signInAndRetrieveDataWithCredential:completion:` This method
+    @brief Please use `signInAndRetrieveDataWithCredential:completion:` instead. This method
         doesn't return additional identity provider data.
  */
 - (void)signInWithCredential:(FIRAuthCredential *)credential
-                  completion:(nullable FIRAuthResultCallback)completion;
+                  completion:(nullable FIRAuthResultCallback)completion
+                      DEPRECATED_MSG_ATTRIBUTE("signInWithCtedential: is deprecated. Please use `si"
```

Comment:
signInWithCredential not signInWithCtedential
Split the line before the method name starts after "Please use ....